odw/services/data-warehouse/app/utils/llm_service.py

```diff
@@ -12,25 +14,92 @@ class LLMService:
     """
     
     def __init__(self):
-        """Initialize the LLM service with Anthropic client."""
+        """Initialize the LLM service with Anthropic client using .env file configuration."""
+        # Check for .env file existence in the data-warehouse directory
+        # Get the directory where this file is located, then go up to data-warehouse root
+        current_dir = Path(__file__).parent  # app/utils/
+        data_warehouse_dir = current_dir.parent.parent  # go up two levels to data-warehouse/
+        env_file = data_warehouse_dir / '.env'
+        
+        cli_log(CliLogData(
+            action="LLMService",
+            message=f"🔍 Looking for .env file at: {env_file}",
+            message_type="Info"
+        ))
+        cli_log(CliLogData(
+            action="LLMService",
+            message=f"🔍 .env file exists: {env_file.exists()}",
+            message_type="Info"
+        ))
+        
+        if not env_file.exists():
+            cli_log(CliLogData(
+                action="LLMService",
+                message="❌ .env file not found. LLM features will be disabled.",
+                message_type="Error"
+            ))
+            cli_log(CliLogData(
+                action="LLMService",
+                message="📝 To enable LLM processing:",
+                message_type="Info"
+            ))
+            cli_log(CliLogData(
+                action="LLMService",
+                message=f"   1. Copy env.example to .env: cp {data_warehouse_dir}/env.example {data_warehouse_dir}/.env",
+                message_type="Info"
+            ))
+            cli_log(CliLogData(
+                action="LLMService",
+                message="   2. Edit .env and set your ANTHROPIC_API_KEY",
```

Comment:
What should the user do after starting moose, seeing this error they're missing .env, then adding .env? Do they restart moose for it to take effect?